app/src/main/index.js

```diff
@@ -328,6 +328,28 @@ function consistentConfigDir (appVersionPath, genesisPath, configPath, gaiaVersi
     exists(gaiaVersionPath)
 }
 
+// check if baseserver is initialized as the configs could be corrupted
+// we need to parse the error on initialization as there is no way to just get this status programmatically
```

Comment:
could we ask gaia to return the status programatically?

Reply:
in theory this would be possible. but I think it doesn't have too high priority. Should I create an issue?

Reply:
yeah, create an issue. just worries me a bit to depend on parsing the error.